src/devices/Ak8963/Register.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ak8963
+{
+    /// <summary>
+    /// Registers of the AK8963. This class is public as 
+    /// need to be accessed when this device is embedded into another one
+    /// like in the MPU9250
+    /// </summary>
+    public enum Register
```

Comment:
internal?

Reply:
Need to be public to be able to be access thru another element like in the case of the MPU9250.

Reply:
Changed all to have it not public, just internal